### DISCUSSION TITLE
no change in tab colors

### DISCUSSION THREAD

**Discussion by jahlbornG** [dbeaver/dbeaver#16778] [General]
so #15760 was supposedly fixed in 22.1.0 (making the tabs use the same color as the connection), however, i'm seeing no difference in tab color.  has anyone else seen a change in tab colors with 22.1.0?

**Comment by LonwoLonwo** [comment]
Hello @jahlbornG 

https://github.com/dbeaver/dbeaver/issues/15760 fixed another problem with the color of SQL Editor elements, yes.

You can create new ticket about tabs coloring.

**Reply by jahlbornG** [reply]
do you know what that ticket actually fixed?  the description for the ticket was about changing the tab color, so it seems weird that the tab color wasn't changed?

**Reply by LonwoLonwo** [reply]
Another problem was fixed in 15760. 
As I said, you can create new ticket for tabs and we will try to fix it again later.
